### ISSUE TITLE
save model and predict 

### ISSUE THREAD

**Issue by lvwuyunlifan** [jeong-tae/RACNN-pytorch#12]
Hello, @jeong-tae:
    I try use you write code of RACNN-pytorch to train my dataset. I want to save train-model. But I have not idea to do this. Can you give me some suggest or idea for save-model and predict. thanks.

**Comment by lvwuyunlifan** [comment]
Is the RACNN model saved in the same way as other classifition models, such as the resnet and inception models with pytorch? I am not sure, after all, it has APN modules and a lot of scales.

**Comment by jeong-tae** [comment]
oh, i didn't notice that there is no save code for model. The training doesn't going well... so I didn't add it.
If you want to save model, please add some code in the train both train() and pretrainAPN to employ.

**Comment by lvwuyunlifan** [comment]
Do you mean that train() and pretrainAPN are saved separately? I see the original papers that are eventually merged together. Can you try to keep them together? I want to predict the test set and take the saved model directly to predict it.

**Comment by jeong-tae** [comment]
sorry, i was confused. you can save net.parameters at once for both network.
RACNN model is built together for both training.
If you want to predict something, just use code like training function.

**Comment by lvwuyunlifan** [comment]
I see that, thanks.
